XamlControlsGallery/ItemPage.xaml

```diff
@@ -151,14 +155,35 @@
                         <RowDefinition Height="Auto" />
                         <RowDefinition Height="Auto" />
                         <RowDefinition Height="Auto" />
+                        <RowDefinition Height="Auto" />
                     </Grid.RowDefinitions>
                     <Grid.ColumnDefinitions>
                         <ColumnDefinition />
                         <ColumnDefinition Width="20" />
                         <ColumnDefinition />
                     </Grid.ColumnDefinitions>
 
-                    <StackPanel x:Name="DocumentationPanel" Grid.ColumnSpan="3" Margin="0,0,0,40">
+                    <StackPanel x:Name="SourcePanel" Grid.Row="0" Grid.ColumnSpan="3" Margin="0,0,0,40">
+                        <TextBlock
+                            x:Name="SourceHeader"
+                            x:Uid="SourceHeader"
+                            Style="{ThemeResource SubtitleTextBlockStyle}"
+                            Text="Sample source code"/>
```

Comment:
Per my other comment, I think there is some ambiguity in the word "sample". This links out to the page's source code itself, which _isn't_ the sample. It's the page that holds the samples. 

Brainstorming: thoughts on changing this text to "View this page on GitHub"? Or maybe "View on GitHub" and the links would say "C# source" and "Xaml source" or something?

Reply:
How about "View page on GitHub" and then have links: "C#/XAML source code" ? That way it is clear where those link directs to.